releasenotes/notes/rename-kubelet-corecheck-8f28eff881fde78a.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Rename ``kubelet`` core check and change metrics prefix to 
```

Comment:
Maybe it would be better to say something like:
Rename ``kubelet_core`` check and change metrics prefix from ``kubernetes_core`` to ``kubernetes`` to allow as replacement for Python ``kubelet`` check.

My thinking here is that no one is currently using the `kubelet_core` check, and the last thing we want is to confuse people looking through the changelog for changes that could impact them

Reply:
good point will update!